sdk/tests/deqp/data/gles3/shaders/linkage.test

```diff
@@ -3492,9 +3494,9 @@ group uniform "Uniform linkage"
 	end
 
 	group block "Uniform blocks"
-		case precision_mismatch
+		case differing_precision
 			version 300 es
-			expect link_fail
+			expect build_successful
```

Comment:
I think WebGL 2 should require this as a link failure case. WebGL 1 requires uniforms precision to match.